dataplane/ebpf/src/main.rs

```diff
@@ -36,8 +36,12 @@ use utils::{ETH_HDR_LEN, ETH_P_IP, IPPROTO_ICMP, IPPROTO_TCP, IPPROTO_UDP};
 // -----------------------------------------------------------------------------
 
 #[map(name = "BACKENDS")]
-static mut BACKENDS: HashMap<BackendKey, Backend> =
-    HashMap::<BackendKey, Backend>::with_max_entries(128, 0);
+static mut BACKENDS: HashMap<BackendKey, BackendList> =
+    HashMap::<BackendKey, BackendList>::with_max_entries(128, 0);
```

Comment:
Let's define another constant for this, that way we can just change it once if we ever need to change it, e.t.c.